sdks/python/apache_beam/io/hadoopfilesystem_test.py

```diff
@@ -36,14 +37,11 @@ class FakeFile(io.BytesIO):
   """File object for FakeHdfs"""
   __hash__ = None  # type: ignore[assignment]
 
-  def __init__(self, path, mode='', type='FILE'):
+  def __init__(self, path, mode='', type='FILE', mtime=None):
     io.BytesIO.__init__(self)
-
-    self.stat = {
-        'path': path,
-        'mode': mode,
-        'type': type,
-    }
+    if mtime is None:
+      mtime = int(time.time() * 1000)
+    self.stat = {'path': path, 'mode': mode, 'type': type, 'mtime': mtime}
```

Comment:
would it be possible to call it `time_ms`? or is `mtime` a property of hadoop file systems somedwhere?